frappe/core/doctype/docfield/docfield.json

```diff
@@ -186,6 +186,7 @@
   },
   {
    "default": "0",
+   "depends_on": "eval:!in_list(['Section Break', 'Column Break', 'Tab Break', 'Table'], doc.fieldtype)",
```

Comment:
Tab is understandable but other field types work in dialog boxes so why don't they work in quick entry form?

Reply:
Go ahead, let's hide table until it's not supported 👍